auth.proto

```diff
@@ -116,10 +116,10 @@ message DeletePolicyReq {
 }
 
 message DeletePoliciesReq {
-  repeated DeletePolicyReq deletePoliciesReq = 1;
+  repeated DeletePolicyFilterReq deletePoliciesReq = 1;
 }
```

Comment:
Lets have seperate message for `DeletePoliciesReq`, `DeletePolicyReq` is different from `DeletePolicyFilterReq`
In `DeletePolicies` all fields are required , but in `DeletePolicyFilter` not all the fields are required. 
`DeletePolicyFilter` filters the relation and then delete. 


```suggestion
message DeletePolicyReq {
  string domain = 1;
  string subject_type = 2;
  string subject_relation = 3;
  string subject_kind = 4;
  string subject = 5;
  string relation = 6;
  string permission = 7;
  string object = 8;
  string object_kind = 9;
  string object_type = 10;
}

message DeletePoliciesReq {
  repeated DeletePolicyReq DeletePolicyReq = 1;
}
```

Reply:
I clarification on this is needed. Should I create a a separate `DeletePolicyFilterRes` and `DeletePolicyFilterReq` to be used?

Reply:
Yes